App.js

```diff
@@ -8,13 +14,13 @@ import * as Amplitude from 'expo-analytics-amplitude';
 import { SafeAreaProvider } from 'react-native-safe-area-context';
 import AppNavigator from './navigation/AppNavigator';
 
-// init with the dev app https://analytics.amplitude.com/echo-church-app-dev
 Amplitude.initialize('915eae61254cd5eaec60df02369eff57');
```

Comment:
@dereckquock should this be in "Keys.js"?

Reply:
yeah 👍  let's put it there